File: cloud-common/src/main/java/com/sequenceiq/cloudbreak/service/CloudbreakServiceException.java
package com.sequenceiq.cloudbreak.service;


public class CloudbreakServiceException extends RuntimeException {
    public CloudbreakServiceException(String message) {
        super(message);
    }

    public CloudbreakServiceException(String message, Throwable cause) {
        super(message, cause);
    }

    public CloudbreakServiceException(Throwable cause) {
        super(cause);
    }
}

File: core/src/main/java/com/sequenceiq/cloudbreak/service/CloudbreakServiceException.kt
package com.sequenceiq.cloudbreak.service


open class CloudbreakServiceException : RuntimeException {
    constructor(message: String) : super(message) {
    }

    constructor(message: String, cause: Throwable) : super(message, cause) {
    }

    constructor(cause: Throwable) : super(cause) {
    }
}
